### PR TITLE
Add ini option for setting additional arguments to a test

### PR DESCRIPTION
**Pull request by elkin**
The update also includes tests for new functionality.

### CODE REVIEW

**Comment by elkin** [comment]
Sometimes it's useful to provide additional arguments to the tests. For example to set log verbosity level.

File: pytest_cpp/plugin.py [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 
+from pytest_cpp.boost import BoostTestFacade
 from pytest_cpp.error import CppFailureRepr, CppFailureError
-
 from pytest_cpp.google import GoogleTestFacade
 
 FACADES = [GoogleTestFacade, BoostTestFacade]
 DEFAULT_MASKS = ('test_*', '*_test')
 
+_ARGUMENTS = 'arguments'
```
Comment (nicoddemus) [inline pytest_cpp/plugin.py]: How about calling this `cpp_arguments` instead? I feel `arguments` is too generic and doesn't indicate well which plugin uses the option.

Comment (elkin) [reply pytest_cpp/plugin.py]: Agree

Comment (elkin) [reply pytest_cpp/plugin.py]: Sure, I'll update README.
Thank you.

Comment (nicoddemus) [reply pytest_cpp/plugin.py]: Thanks!

**Comment by elkin** [comment]
I've updated the pull request.

**Comment by nicoddemus** [comment]
Thanks a lot @elkin!

**Comment by elkin** [comment]
Thank you.

**Pull request merged**